src/librustc/middle/limits.rs

```diff
@@ -1,19 +1,19 @@
-// Recursion limit.
-//
-// There are various parts of the compiler that must impose arbitrary limits
-// on how deeply they recurse to prevent stack overflow. Users can override
-// this via an attribute on the crate like `#![recursion_limit="22"]`. This pass
-// just peeks and looks for that attribute.
+/// Registering limits, recursion_limit, type_length_limit and const_limit
```

Comment:
```suggestion
//! Registering limits, recursion_limit, type_length_limit and const_limit
```

and for all the doc comments below. Otherwise it doesn't document the module but the `use` statement below